docs/index.md

```diff
@@ -87,6 +83,7 @@ EOF
 - `sniff` (Boolean) Set the node sniffing option for the OpenSearch client. Client won't work with sniffing if nodes are not routable.
 - `token` (String) A bearer token or ApiKey for an Authorization header, e.g. Active Directory API key.
 - `token_name` (String) The type of token, usually ApiKey or Bearer
+- `url` (String) OpenSearch URL
```

Comment:
URL not Required anymore?

Reply:
I have updated this in my latest commit peter.